src/Linux/Native/RunAsUser/util.h

```diff
@@ -0,0 +1,60 @@
+/* ****************************************************************************
```

Comment:
Agreed. But not is this change.

---
In reply to: [115851705](https://github.com/Microsoft/RTVS/pull/3516#discussion_r115851705) [](ancestors = 115851705)